sdk/translation/azure-ai-translation-text/src/test/java/com/azure/ai/translation/text/TranslateTests.java

```diff
@@ -202,7 +203,7 @@ public void translateWithProfanity() {
         assertEquals(1, response.getTranslations().size());
         assertEquals("en", response.getDetectedLanguage().getLanguage());
         assertEquals(1, response.getDetectedLanguage().getConfidence());
-        assertTrue(response.getTranslations().get(0).getText().contains("***"));
+        // assertTrue(response.getTranslations().get(0).getText().contains("***")); Created task: https://machinetranslation.visualstudio.com/MachineTranslation/_workitems/edit/164493
```

Comment:
You can use the source text `shit this is fucking crazy shit fuck`.

Reply:
Updated, here are the live test pipeline results: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4098377&view=ms.vss-test-web.build-test-results-tab